NEWS

```diff
@@ -38,6 +38,18 @@ NEWS - user visible changes				-*- outline -*-
    this isn't done anymore so if you need the memory to be initialized
    specify that explicit in the source and recompile
 
+** new -fformat dialect option, and extended SOURCE FORMAT directives, with
+   the following newly supported reference-formats (in addition to
+   FIXED/FREE):
+
+   COBOL85              Fixed-form format with enforcement of Area A (pending)
+   VARIABLE		Microfocus Variable Fixed-form format
```

Comment:
that would be "Micro Focus"; please change that list to be in alphabetical order